source/includes/orders.md

```diff
@@ -81,6 +84,7 @@ special_instructions | Text that indicates any special instructions for the orde
 service_type | Optional, indicates the how the order will be packaged. Possible values: ```dine_in```, ```take_out```. Defaults to ```take_out```.
 channel | Optional, indicates the channel making the request
 total | Order amount in cents
+authorization | Information related to any customer authorization needed to access the order
```

Comment:
optional right? Like they don't need to submit an empty authorization block right?

Reply:
oh good call. I'll update.